curriculum/challenges/spanish/04-data-visualization/json-apis-and-ajax/change-text-with-click-events.spanish.md

```diff
@@ -18,7 +18,7 @@ localeTitle: Cambiar texto con click eventos
 ```yml
 tests:
   - text: Su código debe usar el método <code>document.getElementsByClassName</code> para seleccionar el elemento con <code>message</code> clase y establecer su <code>textContent</code> de <code>textContent</code> en la cadena dada.
-    testString: 'assert(code.match(/document\.getElementsByClassName\(\s*?("|")message\1\s*?\)\[0\]\.textContent\s*?=\s*?("|")Here is the message\2/g), "Your code should use the <code>document.getElementsByClassName</code> method to select the element with class <code>message</code> and set its <code>textContent</code> to the given string.");'
+    testString: 'assert(code.match(/document\.getElementsByClassName\(\s*?("|")message\1\s*?\)\[0\]\.textContent\s*?=\s*?("|")Aquí está el mensaje\2/g), "Tu código debe usar el método <code>document.getElementsByClassName</code> para seleccionar los elementos de la clase <code>message</code> y establecer su <code>textContent</code> al string requerido.");'
```

Comment:
You can not translate the testString portion other than the part between the double quotes ".